src/pharmpy/modeling/pd.py

```diff
@@ -356,3 +348,49 @@ def add_indirect_effect(
     model = set_proportional_error_model(model, dv=2, zero_protection=False)
 
     return model.update_source()
+
+
+def create_baseline_effect(model: Model, expr: str = 'const'):
```

Comment:
Although we are not always consistent with when a function should be named `set_` or `add_` I think this should be named `set_baseline_effect` instead.